### DISCUSSION TITLE
Data table time zone setting

### DISCUSSION THREAD

**Discussion by Commando-Brando** [primefaces/community#1786] [PrimeReact] [answered]
I would like the prime react data table for date types to not automatically convert my databases dates to the users local time. 
I would like to have an options to just compare it not based on locale.

**Comment by melloware** [comment]
You will have to convert the JavaScript dates yourself to be in the Timezone you want them to be in...

**Comment by Commando-Brando** [comment, marked as answer]
Thanks, idk why it was not obvious to do it before putting the data in the table, I was trying to after the fact
